public/js/common.js

```diff
@@ -0,0 +1,141 @@
+// Call Ajax request with @param:
+// $data: data which being send to server
+// $eventTarget: get target button was clicked to handle after request
+// $url: route
+// $method: method request
+// @Return: Call handleAjaxResponse if request successes
+function callAjax($data, $eventTarget, $url, $method) {
+    $.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        }
+    });
+    $.ajax({
+        type: $method,
+        url: $url,
+        data: $data,
+        success: function(data, status) {
+            handleAjaxResponse(data, status, $eventTarget);
+        },
+        error: function(data, status) {
+            //display error
+            handleAjaxResponse(data, status, $eventTarget);
+        }
+    });
+}
+
+// Handle Ajax response with @param:
+// data: response message from server
+// status: response status from server
+// $eventTarget: get target button was clicked to handle
+// @Return: setting something to display
+function handleAjaxResponse(data, status, $eventTarget) {
+    if ($eventTarget.name == "btnConfirm") {
+        $tdHasError = $eventTarget.parentNode.previousSibling.previousSibling;
+        $helpElement = $tdHasError.childNodes[1];
+        if (status == "success") {
+            alert(data['message']);
+            //remove helpblock if exists
+            $tdHasError.className -= " has-error";
+            $helpElement.childNodes[1].disabled = true;
+            $helpElement.childNodes[3].innerHTML = '';
+            $eventTarget.innerHTML = "";
+            $eventTarget.name = "btnEdit";
+            //set date for updated_at
+            $eventTarget.parentNode.previousSibling.
+                                    previousSibling.
+                                    previousSibling.
+                                    previousSibling.
+                                    innerHTML = data[0]['date'].toString().substring(0, 19);
+        } else if (status == "error") {
+            $helpElement.childNodes[3].innerHTML = JSON.parse(data.responseText).quantity[0];
+            $tdHasError.className += " has-error"
+        }
+    } else if ($eventTarget.name == "btnDel") {
+        if (status == "success") {
+            //delete record in table
+            $eventTarget.parentNode.parentNode.remove();
+            $bodyTable = document.getElementById('bodyTable');
+            $rowsElement = $bodyTable.getElementsByTagName('tr');
+            //check if it has not any record
+            if ($rowsElement.length == 0) {
+                window.location.replace('../daily-menus');
+            }
+            alert(data['message']);
+        } else {
+            alert(data['error']);
+        }
+    }
+}
+
+window.onload = function() {
```

Comment:
i have not ever use JQuery before, so i have not enough knowledge to use it current time..